<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManagerDialogBase</class>
 <widget class="QDialog" name="TaskManagerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>536</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>12</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>561</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task_Manager</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="TSlabel">
      <property name="text">
       <string>Task Status:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="CBlabel">
      <property name="text">
       <string>Created By:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBoxTaskStatus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="createByComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>50</number>
      </property>
      <item>
       <property name="text">
        <string>--Select-User--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>robel_recs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mael</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Of Change</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="propertiesTable">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>261</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTaskareaButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Task Area</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadfTaskButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noTaskButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>500</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>No Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentTaskLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Task ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="taskIDText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date:    </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd hh-mm-ss</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="taskListTable">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>241</y>
     <width>531</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
